File: JS CORE/JavaScript Fundamentals/6.Arrays and Matrices/Matrix.js
var matrix = [
    ['0,0', '0,1', '0,2'],
    ['1,0', '1,1', '1,2'],
    ['2,0', '2,1', '2,2']
];

console.log(
    matrix.map(row => row.join(' '))
        .join('\n'));

File: Java/Java Advanced/Java Advanced/Multidimensional Arrays/src/Lab/Matrix.java
package Lab;

public class Matrix {
    public static void main(String[] args) {
        int[][] array = new int[3][4];
        for (int row = 0; row < array.length; row++) {
            for (int col = 0; col < array[0].length; col++) {
                array[row][col] = row + col;
                System.out.print(array[row][col]);
            }
            System.out.print("\n");
        }
    }
}
